jsconfig.json

```diff
@@ -2,5 +2,5 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["src", "test"]
+  "include": ["src"]
```

Comment:
why are we making this change?

Reply:
we don't actually have a `test` folder. So I removed it. This is just for vscode, by the way. https://code.visualstudio.com/docs/languages/jsconfig